scripts/zones/GM_Home/npcs/Linkshell_Concierge.lua

```diff
@@ -0,0 +1,55 @@
+-----------------------------------
+-- Area: GM_Home
+-- NPC: Linkshell_Concierge
+-----------------------------------
+package.loaded["scripts/zones/GM_Home/TextIDs"] = nil
+-----------------------------------
+require("scripts/zones/GM_Home/TextIDs")
+require("scripts/globals/concierge")
+require("scripts/globals/settings")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    -- no real way to know what type these would on retail, so I just same dummy setting for all 3
+    LinkShellConciergeEventTrigger(player,npc,0,dsp.lsconciergetype.EVERYTHING)
+end
+
+function onEventUpdate(player,csid,option)
+    LinkShellConciergeEventUpdate(player,csid,option)
+end
+
+function onEventUpdate(player,csid,option)
+    local lsCmd = option % 0x00000008 
+    local res = LinkShellConciergeEventUpdate(player,csid,option)
+    if lsCmd == 5 and res == 98 then -- dummy value for error
+        player:messageSpecial(ITEM_CANNOT_BE_OBTAINED,0x203)
```

Comment:
add in decimal form to the zones textIDs.lua instead of leaving the raw message ID in file like this. these shift in most client updates and it'll break until we run the textIDs update batch.

Reply:
derp I dunno why I read that backwards ignore me